### DISCUSSION TITLE
How to measure cold start time?

### DISCUSSION THREAD

**Discussion by tomelliot** [vercel/community#7402] [Help]
### Summary

I'm trying to improve user experience for a Next.js 13 app I have running on Vercel. I'm manually logging various metrics in the codebase, but I'm wondering if cold start time for serverless functions is a problem, so I would like to start tracking it.

I have Betterstack as a log dump. Previously I was using another tool which automatically extracted the cold start times from Vercel's logs. I've had a look at the logs being dumped to Betterstack and I haven't found any log entries that are obviously what I need to track.

How would I start tracking cold start times in Betterstack?

### Steps to Reproduce

NA

**Comment by pawlean** [comment]
Hi @tomelliot,

Do any of these guides help you with what you're trying to do?

- https://betterstack.com/community/guides/logging/platforms/vercel-logging/
- https://betterstack.com/docs/logs/vercel/automatic-integration/
- https://betterstack.com/docs/logs/javascript/nextjs/

[a comment by an automated account is not shown]
